cmd/eksctl-anywhere/cmd/createcluster.go

```diff
@@ -221,16 +220,11 @@ func (cc *createClusterOptions) directoriesToMount(clusterSpec *cluster.Spec, cl
 	}
 
 	if clusterSpec.Config.Cluster.Spec.DatacenterRef.Kind == v1alpha1.CloudStackDatacenterKind {
-		env, found := os.LookupEnv(decoder.EksaCloudStackHostPathToMount)
-		if found && len(env) > 0 {
-			mountDirs := strings.Split(env, ",")
-			for _, dir := range mountDirs {
-				if _, err := os.Stat(dir); err != nil {
-					return nil, fmt.Errorf("invalid host path to mount: %v", err)
-				}
-				dirs = append(dirs, dir)
-			}
+		extraMounts, err := decoder.GetCloudStackExtraMounts()
+		if err != nil {
+			return nil, fmt.Errorf("invalid host path to mount: %v", err)
```

Comment:
this error message may be confusing. We're not doing any validation on the host path itself, right? I feel like this more represents an issue with retrieving the extra mounts

Reply:
GetCloudStackExtraMounts function checks if the host paths are existing. If one of the host paths doesn't exist the function returns an error.